flyteidl/protos/flyteidl/service/dataproxy.proto

```diff
@@ -63,6 +63,10 @@ message CreateUploadLocationRequest {
   // it will force clients to add this metadata to the object.
   // This make sure dataproxy is backward compatible with the old flytekit.
   bool add_content_md5_metadata = 7;
+
+
+  // Optional, org key applied to the resource.
+  string org = 8;
```

Comment:
qq: do we need `org` for CreateDownloadLocationRequest?

Reply:
nope! it's already in the identifier in the [source](https://github.com/flyteorg/flyte/blob/master/flyteidl/protos/flyteidl/service/dataproxy.proto#L111)

Reply:
and I figured it's not worth touching CreateDownloadLocationRequest since it's [deprecated](https://github.com/flyteorg/flyte/blob/master/flyteidl/protos/flyteidl/service/dataproxy.proto#L70)